signac/__main__.py

```diff
@@ -759,19 +755,19 @@ def main_config_verify(args):
         cfg = config.read_config_file(config.USER_CONFIG_FN)
     else:
         cfg = config.load_config()
-    if cfg is None:
-        if args.local and args.globalcfg:
-            mode = " local or global "
-        elif args.local:
+    if not cfg:
+        if args.local:
```

Comment:
If you look up 10 lines to 749 you'll see that is an error case. This block was always unreachable.

Reply:
Also, just so it's clear, this won't change the case where a user just specified local or globalcfg. If the user specified _neither_ however, the expected message is actually that neither file was found (because we search both by default), not an empty string (as was previously shown).